texts/puupankki/puupankki.kaz.conllu

```diff
@@ -4256,20 +4622,21 @@
 6	,	,	PUNCT	cm	_	10	punct	_	_
 7	қала	қала	NOUN	n	Case=Nom	9	compound	_	SpaceAfter=No
 8	-	-	PUNCT	guio	_	9	punct	_	SpaceAfter=No
-9	қалаға	қала	NOUN	n	Case=Dat	10	nmod	_	_
+9	қалаға	қала	NOUN	n	Case=Dat	10	obl	_	_
 10	орнады	орна	VERB	v	Mood=Ind|Number=Plur|Person=3|Tense=Past|VerbForm=Fin	0	root	_	SpaceAfter=No
 11	.	.	PUNCT	sent	_	10	punct	_	_
 
 # sent_id = kdt.tagged.txt:35:570
```

Comment:
[Line 4647 Sent kdt.tagged.txt:35:570 Node 16]: [L3 Syntax orphan-parent] The parent of 'orphan' should normally be 'conj' but it is 'nsubj'.